test/appCenterConfig.ts

```diff
@@ -28,19 +28,24 @@ describe('AppCenterConfig', function () {
         pathToAndroidConfig = path.join(root, "android", "app", "src", "main", "assets", "appcenter-config.json");
 
         // Cleanup the files.
-        fs.writeFileSync(pathToAppCenterConfigPlist, "");
-        fs.writeFileSync(pathToMainPlist, "");
+        fs.writeFileSync(pathToAppCenterConfigPlist, '<?xml version="1.0" encoding="UTF-8"?>');
```

Comment:
Please replace `fs` with `mock-fs`, you should avoid of using real fs while implementing unit tests.

Reply:
how do I test saving values then?

Reply:
nevermind I can